### ISSUE TITLE
[Question] Are you interested in a CodeBlocks action?

### ISSUE THREAD

**Issue by DPSLynix** [premake/premake-core#616]

**Comment by tvandijck** [comment]
Yes, we are always interested in additional modules, and codeblocks is one that others have been asking for as well... The best would be to make a module out of it, so it can be easily imported and used by anyone, without modifying premake itself... there is tons of modules out there already, so plenty of examples:
https://github.com/premake/premake-core/wiki/Developing-Modules

**Comment by DPSLynix** [comment]
I'll do that whenever I got time. 

Do you have any request or recommendation regarding the source code of my port?

**Comment by tvandijck** [comment]
No, it looks pretty good in my opinion, and already looks like it's mostly setup as a module, so it shouldn't be too much work.

**Comment by chris-be** [comment]
Hi,

Thanks to your work I was able to use Code::Blocks for my needs :)
I only took the pieces, created a "module" and added "clang".
Please visit the [git repo](https://github.com/chris-be/premake-codeblocks) and tell me who was implied to adjust authors info and what to put in LICENSE.txt.

Thanks !

**Comment by starkos** [comment]
Nice! Can you update [the Modules page](https://github.com/premake/premake-core/wiki/Modules) with a link to your repository? Also, I guess we should remove the link from the "Built-In" section, since we no longer ship a C::B module by default?

**Comment by chris-be** [comment]
Done. Note that there is no "C::B" link in the Built-In section (in this page at least).

**Comment by S6066** [comment]
It would be interesting to put Code::Blocks directly into the built-in modules, as it is a well used IDE
Also, the link for CMake in the [Modules page](https://github.com/premake/premake-core/wiki/Modules) is broken
